tests/forms_tests/tests/test_media.py

```diff
@@ -520,3 +520,24 @@ def test_html_safe(self):
         media = Media(css={'all': ['/path/to/css']}, js=['/path/to/js'])
         self.assertTrue(hasattr(Media, '__html__'))
         self.assertEqual(force_text(media), media.__html__())
+
+
+@override_settings(
+    STATIC_URL='http://media.example.com/static/',
+    INSTALLED_APPS=('django.contrib.staticfiles', ),
+    STATICFILES_STORAGE='forms_tests.contrib.storages.StaticTestStorage'
```

Comment:
We like to include a trailing comma in kwargs like this so if more items are added, we don't have to modify this line again.